### ISSUE TITLE
Graphics.DrawLine

### ISSUE THREAD

**Issue by gio3k** [Facepunch/sbox-issues#2630]
### For?

S&Box

### What can't you do?

Easily draw a 2D line (preferably with the ability to make it cubic like GraphicsItem.CubicLineTo)

Drawing UI lines seems to be hell, especially while there's no SVG support. This would help for the time being

### How would you like it to work?

Graphics.DrawLine, maybe also Graphics.DrawCubicLine

### What have you tried?

using Render, didn't work and didn't have what I needed anyways

**Comment by matekdev** [comment]
There were previously a bunch of different `Graphics.DrawX` functions before Garry did the rework. It'd be nice if they were brought back to the current API.

**Comment by xezno** [comment]
Agreed, it would be awesome if these were brought over into the base s&box API.. for now, I've moved them over myself into this: https://gist.github.com/xezno/c890c479e32fb314f0c0cd7739afd6ea; feel free to use it if you want

**Comment by anthonysharpy** [comment]
you can use Graphics.Draw to make lines but yeah it's not great atm, especially since there's no obvious way to adjust thickness etc
